EIPS/eip-token-standard.md

```diff
@@ -0,0 +1,170 @@
+## Preamble
+
+    EIP: 20
+    Title: ERC-20 Token Standard
+    Author: Fabian Vogelsteller <fabian@ethereum.org>, Vitalik Buterin <vitalik.buterin@ethereum.org>
+    Type: Standard
+    Category: ERC
+    Status: Accepted
+    Created: 2015-11-19
+
+
+## Simple Summary
+
+A standard interface for tokens.
+
+
+## Abstract
+
+The following standard allows for the implementation of a standard API for tokens within smart contracts.
+This standard provides basic functionality to transfer tokens, as well as allow tokens to be approved so they can be spent by another on-chain third party.
+
+
+## Motivation
+
+A standard interface allows any tokens on Ethereum to be re-used by other applications: from wallets to decentralized exchanges.
+
+
+## Specification
+
+## Token
+### Methods
+
+**NOTE**: Callers should handle `false` from `returns (bool success)`.  Callers should not assume that `false` is never returned!
```

Comment:
I'll be making suggestions to make the document clearer per RFC 2119.

should handle `false` -> MUST handle
should not assume that `false` -> MUST NOT assume